farm/modeling/adaptive_model.py

```diff
@@ -256,7 +256,6 @@ def __init__(
         self.lm_output_types = (
             [lm_output_types] if isinstance(lm_output_types, str) else lm_output_types
         )
-        assert len(self.lm_output_types) == len(self.prediction_heads)
```

Comment:
Why are we getting rid of this assertion here?

Reply:
It fails during embedding extraction like described in #397